openapi/broadcasts-api.yaml

```diff
@@ -1278,23 +1278,23 @@ components:
           title: Phq Attendance
         phq_rank:
           description: |-
-            The PHQ Rank number of the physical event. PHQ Rank represents the physical event’s impact independent of its geographical location.
+            The PHQ Rank number of the physical event. PHQ Rank represents the physical event's impact independent of its geographical location.
 
 
             E.g. `100`
           type: integer
           title: Phq Rank
         local_rank:
           description: |-
-            The Local Rank number of the physical event. Local Rank represents the physical event’s impact on its local geographical location.
+            The Local Rank number of the physical event. Local Rank represents the physical event's impact on its local geographical location.
 
 
             E.g. `100`
           type: integer
           title: Local Rank
         aviation_rank:
```

Comment:
This field needs to be marked deprecated - must have been missed when we marked other Aviation Rank fields deprecated.